geode-docs/developing/data_serialization/PDX_Serialization_Features.html.md.erb

```diff
@@ -44,19 +45,10 @@ If old clients write entries of a PDX type they know but the central registry do
 
 To avoid this problem, clients may be configured with the system property in the table below to clear their PDX type cache when they disconnect from the cluster. After clearing their cache, old clients will re-generate type information for all PDX types, including the types the central registry "forgot". Since new PDX type information will be written in the central registry before entries of that type are written in the cluster, the central registry and all clients, old and new, will store consistent PDX type information.
 
-| Name                                      | Description                                             | Default |
-|-------------------------------------------|---------------------------------------------------------|---------|
-| `gemfire.ON_DISCONNECT_CLEAR_PDXTYPEIDS`  | Clear pdxType ids when client disconnects from servers. | `false` |
--   In the API, set the System properties before the cache creation call. Example:
-
-    ``` pre
-    System.setProperty(PoolImpl.ON_DISCONNECT_CLEAR_PDXTYPEIDS, "true");
-    ```
--   At the java command line, pass in System properties using the -D switch. Example:
-
-    ``` pre
-    java -Dgemfire.ON_DISCONNECT_CLEAR_PDXTYPEIDS=true
-    ```
+| Name                                      | Default | Applies to (Java Client, Native Client)|
```

Comment:
Suggestion: instead of "Applies to (Java Client, Native Client" use "Client type".

Reply:
done.